resources/dex/templates/dex-route-rule.yaml

```diff
@@ -1,11 +1,15 @@
 
-apiVersion: config.istio.io/v1alpha2
-kind: RouteRule
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
 metadata:
   name: dex-cors-route-rule
 spec:
```

Comment:
Not related to task but since you have made a change here, please rename it to dex-virtual-service or just dex

Reply:
Actually this file should be removed, there already is dex-virtualservice